src/common/options/rgw.yaml.in

```diff
@@ -4250,3 +4250,19 @@ options:
   flags:
   - startup
   with_legacy: true
+- name: rgw_data_sync_start_full_sync
```

Comment:
>  i don't think we should allow skipping full syncs for normal bucket sync because that subverts disaster recovery and breaks symmetry within the zonegroup

That is the part I still struggle to distinguish a little bit :)
Can you please elaborate a little more? do you mean if the source was within the same zonegroup we should disregard the feature and go for full sync?

Reply:
i'm referring to 'normal bucket sync' where the destination bucket is the same as the source bucket, just on another zone in the zonegroup. the kind you get by default in multisite without any bucket replication policy

i've been arguing that this replication should always be symmetric within the zonegroup, and for deprecation of features like sync policy and 'bucket sync disable' that break this symmetry. but this pr adds a new way to break the symmetry by skipping full sync

Reply:
I moved this to the zonegroup feature. Also i've considered the normal bucket sync to disregard the feature and force the full-sync.